.meteor/packages

```diff
@@ -75,7 +75,7 @@ gadicc:blaze-react-component
 
 # Testing packages
 dburles:factory
-dispatch:mocha
+meteortesting:mocha
```

Comment:
@spencern I think I read that the test driver package no longer needs to be listed here. So I would remove it from here and just make sure the CLI `reaction test` command is updated. My real preference would be:

1. Change the npm "test" script to run the app tests. (Copy the command the CLI currently uses.)
2. Change the CLI `reaction test` command to simply run `npm test`